pkg/store/bucket.go

```diff
@@ -712,19 +712,9 @@ func blockSeries(
 			refs: make([]uint64, 0, len(chks)),
 			chks: make([]storepb.AggrChunk, 0, len(chks)),
 		}
-		for _, l := range lset {
-			// Skip if the external labels of the block overrule the series' label.
-			// NOTE(fabxc): maybe move it to a prefixed version to still ensure uniqueness of series?
-			if extLset[l.Name] != "" {
-				continue
-			}
-			s.lset = append(s.lset, l)
-		}
-		for ln, lv := range extLset {
-			s.lset = append(s.lset, labels.Label{Name: ln, Value: lv})
-		}
-		sort.Sort(s.lset)
 
+		// find is used to check whether there is at least one chunk in the required time range.
+		var find bool
```

Comment:
Should this be named `skipped` or `found`? 
```suggestion
		var skipped bool
```
The comment says:
> find is used to check whether there is at least one chunk in the required time range.

But it's only set the `true` when it's skipped. I'm a bit confused here about the naming. Could we be more clear here?

Reply:
Done